lib.properties

```diff
@@ -1,12 +1,15 @@
-VERSION_NAME=1.1.1
-VERSION_CODE=3
-GROUP=cz.ackee.ui
-BINTRAY_REPO=ui
-BINTRAY_ORGANISATION=ackeecz
+VERSION_NAME=1.1.2
```

Comment:
2.0.0. This release contains breaking changes

Reply:
Updated https://github.com/AckeeCZ/TextInputLayout/pull/12/commits#:~:text=%E2%99%BB%EF%B8%8F,and%20fix%20version